### ISSUE TITLE
Command-click a row or command-drag rows to toggle selection in Tables?

### ISSUE THREAD

**Issue by mbostock** [observablehq/inputs#90]
Ref. https://observablehq.com/@observablehq/input-table#comment-a55f3587d9ddb0f9

This would be especially nice since you don’t have to click the tiny checkboxes.

**Comment by jrus** [comment]
My preferred behavior would be to treat any mouse interactions where the mousedown started in the column of checkboxes as if the table rows were a multi-selection input.

So then click (optionally drag) starting in the checkbox column would result in a selection of the set union between the previous selection and the drag range, and command-click (optionally drag) would select the set difference between the previous selection and drag range.

(This is the behavior in Mac Safari; play with that one with various combinations of clicks, drags, and modifiers to get a feel for it. I am not sure how these inputs behave in other browsers / operating systems.)

**Comment by jrus** [comment]
I agree that clicking and dragging starting outside the checkbox row should be used for text selection (or possibly selection of a rectangle of cells, excel-style).
